components/console/esp_console.h

```diff
@@ -244,9 +242,9 @@ size_t esp_console_split_argv(char *line, char **argv, size_t argv_size);
  *   linenoiseSetCompletionCallback(&esp_console_get_completion);
  *
  * @param buf the string typed by the user
- * @param lc linenoiseCompletions to be filled in
+ * @param lc esp_linenoise_completions_t to be filled in
  */
-void esp_console_get_completion(const char *buf, linenoiseCompletions *lc);
+void esp_console_get_completion(const char *buf, esp_linenoise_completions_t *lc);
```

Comment:
Just to check: with this change, does the `examples/system/console/advanced` example from IDF v5.0 compile okay with this brach?

(I assume the answer is "yes" since you didn't have to modify the example. Is that due to that `#include "esp_linenoise.h" above which you have added?)

Reply:
I've done my testing with the "basic" example. But I think you are correct.